README.md

```diff
@@ -133,4 +133,4 @@ Zenodo DOI:
 
 The code in this repository is licensed under Apache License 2.0, except for the
 dependencies included under `annif/static/css` and `annif/static/js`,
-which have their own licenses. See the file headers for details.
+which have their own licenses. See the file headers for details. Using the optional Yake backend may change the licence of Annif to GPLv3, because [YAKE](https://github.com/LIAAD/yake) is licensed under GPLv3.
```

Comment:
This doesn't sound right...we need to figure out the exact wording to use but importing Yake doesn't really *change* the license of Annif - what it may do is affect the terms under which the combination of Annif and Yake may be used and distributed.